morituri/program/flac.py

```diff
@@ -0,0 +1,19 @@
+from subprocess import check_call, CalledProcessError
+
+import logging
+logger = logging.getLogger(__name__)
+
+FLAC = 'flac'
+
+
+def encode(infile, outfile):
+    """
+    Encodes infile to outfile, with flac.
+    Uses '-f' because morituri already creates the file.
+    """
+    try:
+        check_call(['flac', '--totally-silent', '--verify', '-o', outfile,
+                    '-f', infile])
```

Comment:
Consider using `--silent` instead of `--totally-silent` and capturing any errors written (`stderr=subprocess.PIPE`). Would be much more helpful to know that flac failed with a specific error, rather than just the fact that it failed.